src/main/java/seedu/contax/model/Model.java

```diff
@@ -115,7 +118,9 @@
      */
     void setTag(Tag target, Tag editedTag);
 
-    ObservableList<Tag> getTagList();
+    void updateFilteredTagList(Predicate<Tag> predicate);
+
+    ObservableList<Tag> getFilteredTagList();
```

Comment:
I would appreciate if you could add javadocs.

Reply:
Fixed in 383c8d1a